testing/it/e2e-selenium-utils/src/main/java/com/adobe/cq/wcm/core/components/it/seljup/util/Commons.java

```diff
@@ -459,6 +460,47 @@ public static String addComponent(CQClient client, String component, String pare
         }
     }
 
+    /**
+     * Adds a component to a page with retires
+     *
+     * @param client            CQClient
+     * @param component          mandatory components resource type
+     * @param parentCompPath     mandatory absolute path to the parent component
+     * @param nameHint           optional hint for the component nodes name, if empty component name is taken
+     * @param order              optional where to place component e.g. 'before product_grid', if empty, 'last' is used
+     * @param timeout            try retries for this time duration
+     * @param delay              interval between successive retries
+     * @throws ClientException
+     */
+
+    public static String addComponentWithRetry(final CQClient client, final String component, final String parentCompPath, final String nameHint,
+                                               final String order, long timeout, long delay, final int... expectedStatus) throws ClientException {
+
+        class AddComponentPolling extends Polling {
+            String componentPath;
+
+            AddComponentPolling() {
+            }
+
+            public Boolean call() throws Exception {
+                this.componentPath = addComponent(client, component, parentCompPath, "text", null);
```

Comment:
I think it should be addComponent(client, component, parentCompPath, nameHint, order, expectedStatus);

cc: @vladbailescu